pkg/infra/kubernetes/plugin.go

```diff
@@ -150,14 +161,16 @@ func (p *Kubernetes) getKlothoCharts(constructGraph *core.ConstructGraph) (map[s
 			}
 		}
 
-		if cfg.HelmChartOptions.Install {
+		if cfg.Type == "kubernetes" {
```

Comment:
could you put a type somewhere like in config for this? We do it in aws, but my feeling is that aws may need to depend on our kubernetes library so maybe rather than the Kubernetes type being in the aws provider config, it should be in here

Reply:
I just added exec unit types as consts in config. Feel free to adjust as desired.